docs/curation-review-tut.md

```diff
@@ -0,0 +1,417 @@
+# Cellxgene Curation Tutorial
+
+<!--- ![image](https://user-images.githubusercontent.com/25663501/111377133-b07a4c80-8676-11eb-8eb8-07ca4d7a77e9.png) --->
+
+<img align="right" src="https://user-images.githubusercontent.com/25663501/111377133-b07a4c80-8676-11eb-8eb8-07ca4d7a77e9.png">
+
+
+
+
+<!--- ## Overview --->
+<!--- purpose: Motivate submission to the data portal give context --->
+
+[Cellxgene's publishing platform](https://cellxgene.cziscience.com/) and [interactive single cell data explorer](https://github.com/chanzuckerberg/cellxgene) is a system which is optimized for easy access, interrogation and sharing/reuse of single cell data. In order to meet these standards and to facilitate easy navigation/aggregation among the growing number of single cell datasets, the cellxgene publishing platform currently accepts curated anndata objects adhering to a [comprehensive data schema](corpora_schema.md). In this tutorial, you will gain all the essential information for doing the following: 
```

Comment:
navigation/aggregation --> navigation and aggregation

Reply:
adjusted wording